java/Yaroslav-Dzhur/src/com/zoolatech/lecture1/tasks/_9/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.zoolatech.lecture1.tasks._9;
+
+/*
+Write a program that accepts an integer value and outputs all its digits in the reverse order on a new line.
+*/
+
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+
+        System.out.println("Enter number: ");
+        int number = scanner.nextInt();
+        int numberCreate;
```

Comment:
since the variable `numberCreate` is not used outside of the loop, it can be defined inside of it